backend/controllers/event.controller.ts

```diff
@@ -27,50 +31,22 @@ export const createVerifiedEvent = async (
   next: NextFunction,
 ) => {
   try {
-    // Get userId
-    // const userId = req.user.id; // get userId from the request -> should set in auth middleware
     // Validate request id param
     const organizationId = IdParamSchema.parse(req.params).id;
 
     // Validate the Event data with zod schema
-    const validatedEventData = EventCreateSchema.parse(req.body);
-    let newEvent: Event;
+    // Note we are expecting form data on anything that includes a file upload, thus we must parse the data
+    const validatedEventData = EventCreateSchema.parse(
+      JSON.parse(req.body.data),
```

Comment:
This doesn't work for me:
`JSON.parse(req.body.data)`
-> req.body.data is undefined
When processing multipart/form-data with Multer, the text fields are stored in `req.body` and the file is in `req.file`